Sources/Commands/SwiftTestCommand.swift

```diff
@@ -856,20 +814,34 @@ final class TestRunner {
     /// Constructs arguments to execute XCTest.
     private func args(forTestAt testPath: AbsolutePath) throws -> [String] {
         var args: [String] = []
-        #if os(macOS)
-        if library == .xctest {
+#if os(macOS)
+        switch library {
+        case .xctest:
             guard let xctestPath = self.toolchain.xctestPath else {
                 throw TestError.xcodeNotInstalled
             }
-            args = [xctestPath.pathString]
-            args += additionalArguments
-            args += [testPath.pathString]
-            return args
+            args += [xctestPath.pathString]
+        case .swiftTesting:
+            // FIXME: better way to get path to self
+            let toolPath = String(unsafeUninitializedCapacity: 2048) { buffer in
+                var count = UInt32(buffer.count)
+                _NSGetExecutablePath(buffer.baseAddress!, &count)
```

Comment:
Is there a more idiomatic way to get the current process' path in SwiftPM I should use? This is idiomatic on Darwin, but pretty low-level (and I haven't attempted to handle arbitrarily long paths.)

Actually, since `swift-test` is probably _not_ the final home for the Swift Testing host executable (for one thing, it links to SwiftSyntax which causes runtime noise), it is probably moot?